lib/kubernetes-deploy/runner.rb

```diff
@@ -113,11 +116,15 @@ def run(verify_result: true, allow_protected_ns: false, prune: true)
       end
 
       if verify_result
-        deploy_resources(resources, prune: prune, verify: true)
+        ::StatsD.measure('duration.normal_resources', tags: %W(namespace:#{@namespace} sha:#{ENV['REVISION']})) do
+          deploy_resources(resources, prune: prune, verify: true)
+        end
         record_statuses(resources)
         success = resources.all?(&:deploy_succeeded?)
       else
-        deploy_resources(resources, prune: prune, verify: false)
+        ::StatsD.measure('duration.normal_resources', tags: %W(namespace:#{@namespace} sha:#{ENV['REVISION']})) do
```

Comment:
When we start deploying shopify core to multiple clusters (contexts), all of them would be deployed to the same namespace ("shopify-core" or whatever). From what I see now that would result to mix up of metrics.
How do you think we should solve it? We could also include `context` into the tags, however I'm not sure if it's the best idea.